lib/orderly.rb

```diff
@@ -24,5 +25,19 @@ def html_for_node(node)
         page.driver.evaluate_script("arguments[0].innerHTML", node.native)
       end
     end
+
+    def text_for_node(node)
+      text = begin
```

Comment:
I think this complexity is unwarranted for text, because capybara provides an API for that. You can just do `node.text`